Gemfile.lock

```diff
@@ -432,6 +432,3 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   web-console (~> 2.0)
   webmock
-
-BUNDLED WITH
```

Comment:
@timurvafin, please do `gem update bundler && rbenv rehash` to update bundler to 1.10.6, so we all will be on the same version.